.taskcluster.yml

```diff
@@ -9,8 +9,10 @@ tasks:
     decision_task_id: {$eval: as_slugid("decision_task")}
     expires_in: {$fromNow: '1 year'}
 
-    # We may need to change the following line because of https://github.com/mozilla-releng/scriptworker/issues/326
-    user: ${event.sender.login}
+    user:
+      $if: 'tasks_for == "github-push"'
+      then: ${event.pusher.name}
+      else: ${event.sender.login}
```

Comment:
[The docs that I found](https://developer.github.com/v3/activity/events/types/) mentioned that `sender` and `pusher` are the same person.

> Sender and pusher are the same user who initiated the push event, but the sender object contains more detail.

Just curious: how does this resolve the CoT breakage?

Reply:
Oh, is this because we're using `event.pusher.name` elsewhere, and this makes it consistent?
How was CoT getting two different values in [this build](https://taskcluster-artifacts.net/XJd4B25rSAG44qEGJXAFvQ/0/public/logs/chain_of_trust.log)?

Reply:
Excellent call! I totally missed that message. Thanks to you, I found the root cause: https://github.com/mozilla-releng/scriptworker/blob/bc321ccc9f17c6e7cb0dc395529ecd0462fa861b/scriptworker/cot/verify.py#L1186 calls https://api.github.com/repos/mozilla-mobile/reference-browser/commits/e9347cd1ac84e0c1f7f1a3f1fc0c561b5c9175d4. https://github.com/mozilla-releng/scriptworker/blob/bc321ccc9f17c6e7cb0dc395529ecd0462fa861b/scriptworker/cot/verify.py#L1205 should use `commiter.login` instead of `author.login`.